src/structures/Guild.js

```diff
@@ -175,6 +175,12 @@ class Guild extends Base {
      */
     this.joinedTimestamp = data.joined_at ? new Date(data.joined_at).getTime() : this.joinedTimestamp;
 
+    /**
+     * The shard ID that this guild is on
+     * @type {number}
+     */
+    this.shardId = data.shard ? data.shard.id : 0;
```

Comment:
should be `?number` and default to `null` not `0`

Reply:
I literally had it like that but then @KingDGrizzle suggested I changed it to 0, which made more sense to me.